app/renderer/main.html

```diff
@@ -41,4 +41,6 @@
 </body>
 <script src="js/main.js"></script>
 
+<!-- To trigger electron.reload on changes in renderer from gulp -->
+<script>require('electron-connect').client.create()</script>
```

Comment:
As i said in chat. Try this
```javascript
window.addEventListener('load', () => {
  const isDev = require('electron-is-dev');
  if (isDev) {
    require('electron-connect').client.create();
  }
});
```

Reply:
@abhigyank we don't need to run this script in production so please use the code which @cPhost has suggested.